projects/igniteui-angular/src/lib/grids/grid/grid.pipes.ts

```diff
@@ -27,14 +28,15 @@ export class IgxGridSortingPipe implements PipeTransform {
         this.gridAPI = <IgxGridAPIService>gridAPI;
     }
 
-    public transform(collection: any[], expressions: ISortingExpression[], id: string, pipeTrigger: number): any[] {
+    public transform(collection: any[], expressions: ISortingExpression[], sorting: IGridSortingStrategy,
+                     id: string, pipeTrigger: number): any[] {
```

Comment:
I don't know why we still write the `pipeTrigger` as a parameter. It's not in your change, but the pipe trigger can be passed, regardless of whether it's defined in the parameter list.